src/main/java/graphql/execution/DataLoaderDispatchStrategy.java

```diff
@@ -0,0 +1,56 @@
+package graphql.execution;
+
+import graphql.Internal;
+import graphql.schema.DataFetcher;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+@Internal
+public interface DataLoaderDispatchStrategy {
+
+    DataLoaderDispatchStrategy NO_OP = new DataLoaderDispatchStrategy() {
+    };
+
+
+    default void executionStrategy(ExecutionContext executionContext, ExecutionStrategyParameters parameters) {
+
+    }
+
+    default void executionStrategy_onFieldValuesInfo(List<FieldValueInfo> fieldValueInfoList, ExecutionStrategyParameters parameters) {
```

Comment:
xxx_yyyYyyy is weird naming

Whats the point of `executionStrategy_` here - why not `onFieldValuesInfo`